python_modules/dagster/dagster_tests/core_tests/execution_tests/test_context.py

```diff
@@ -405,3 +405,18 @@ def the_op(context: int):
         @asset
         def the_asset(context: int):
             pass
+
+
+def test_get_context():
```

Comment:
might be worth testing thread behavior? I guess ContextVar should handle it fine

Reply:
ya feel pretty good about thread & coroutine concurrency 
https://docs.python.org/3/library/contextvars.html